src/runners/helpers/vault.py

```diff
@@ -28,5 +32,15 @@ def decrypt_if_encrypted(ct: Optional[str] = None, envar: Optional[str] = None)
 
         return res['Plaintext'].decode()
```

Comment:
Do we need to b64decode the plaintext here? When I use `aws kms decrypt` on the CLI, I get plaintext that's base64 encoded, and I need to decode it in order to get useful data out again.

Reply:
Yes -- I think it's the same here as it is on the CLI. In order to protect against funky stuff sneaking into the pt, KMS always b64's it. That way you're also dealing with pt that is bytes, not strings, side-stepping the need for a standard encoding:

    >>> from base64 import b64decode
    >>> b64decode('')
    b''